src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/PortableInfoboxPageObject.java

```diff
@@ -79,9 +87,7 @@ public String getBackgroundColor() {
     return italicElements;
   }
 
-  public List<WebElement> getHeaderElements() {
-    return pInfoTitleH3;
-  }
+  public List<WebElement> getHeaderElements() { return pInfoTitleH3; }
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)